controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -335,17 +334,17 @@ func (r *NodeNetworkConfigurationPolicyReconciler) forceNNSRefresh(name string)
 	nns := &nmstatev1beta1.NodeNetworkState{}
 	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: name}, nns)
 	if err != nil {
-		log.Error(err, "failed retrieving NodeNetworkState to force refresh")
+		log.Error(err, "WARNING: failed retrieving NodeNetworkState to force refresh, it will be refreshed after regular period")
```

Comment:
Again, this should not be seen as `ERROR: WARNING: ` in the logs, that would be misleading.

When somebody opens a BZ and shares nmstate logs. I want to be able to search it for "ERROR" and the first line that pops up must be the cause of the bug.